packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -214,7 +214,7 @@ export class QuoteRequestor {
      * @returns a list of TypedMakerUrl instances
      */
     public static getTypedMakerUrlsAndWhitelist(
```

Comment:
what does "typed" mean here?

Reply:
Discussed in "person": no need to adjust things here